Sources/Vortex/Settings/Settings.swift

```diff
@@ -0,0 +1,314 @@
+//
+// Settings.swift
+// Vortex
+// https://www.github.com/twostraws/Vortex
+// See LICENSE for license information.
+//
+
+import SwiftUI
+
+extension VortexSystem {
```

Comment:
I'd be happy to move this whole struct outside `VortexSystem` and just call it `VortexSettings`; it's a bit wordy as-is.

Reply:
agreed.